lib/apache

```diff
@@ -192,6 +214,12 @@ function enable_apache_site {
         if [[ -f ${enabled_site_file}.disabled ]] && [[ ! -f ${enabled_site_file} ]]; then
             sudo mv ${enabled_site_file}.disabled ${enabled_site_file}
         fi
+    elif is_clearlinux; then
```

Comment:
I don't see any differences here from the is_fedora case, just add || is_clearlinux to line 211?